src/modules/common/DateSelect.js

```diff
@@ -1,8 +1,11 @@
 import React, { Component } from 'react';
+import styled from 'styled-components';
```

Comment:
Not crazy about imported `styled-components` here just to create a class essentially. Maybe @kalongthewires has a better idea?

Reply:
I have no issues with this. I have similarly wrapped third-party components. To me, using styled-components is generally about creating a component instead of a class -> a one-to-one replacement of class names with components. Interested to hear your objections.

Reply:
No, you're right. Having the file size next to my imports makes me think about it a lot more and it felt like I could refactor this in a way that wouldn't require styled-components, but anything else wouldn't be straight forward. I retract my comment.

Reply:
Looking at this again - I do think DateSelect probably doesn't need to be a class. It should probably *only* be a styled-component. At least, setting internal state doesn't give us the data for the preview, that will need to be handled via `props` and Redux. This will probably end up being a stateless styled-component instead. But that can happen in another PR...

Reply:
Completely agree. I'll leave the styling in place since it looks pretty, but create an issue for wiring up the DateSelect that can include anything else it needs.